app/views/clients/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="row">
+    <div class="col-lg-11">
+      <div class="text-right">
+          <%= link_to "Create Client", new_client_path, id: "refresh-btn", class: 'btn btn-primary btn-lg' %>
```

Comment:
refresh-btn should be removed from all links.

Reply:
Removed.